### PR TITLE
Fix unclosed html tags

### PR DESCRIPTION
**Pull request by macdonst**
## Description

We are moving to a new parser for markdown content. It requires html tags to be closed so this PR fixes the instances of `` that should be ``.

## Related Issue

n/a

## Motivation and Context

Sets up the repo to be parsed by our our new internal/external documentation sites.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My change follows the documentation style of this project.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: tutorial/7-sending-notifications.md [hunk 1]
```diff
@@ -95,12 +99,5 @@ This will produce a Microsoft Teams message which looks like this:
 
 <!-- Remix Button -->
 <a href="https://glitch.com/edit/#!/remix/adobe-cloudmanager-api-tutorial-step7-msteams" target="_new">
-  <img src="../img/glitch.png" alt="Remix in Glitch" id="glitch-button">
+  <img src="../img/glitch.png" alt="Remix in Glitch" id="glitch-button"/>
 </a>
-
-<style type="text/css">
```
Comment (justinedelson) [inline tutorial/7-sending-notifications.md:101]: @macdonst why was this removed?

Comment (macdonst) [reply tutorial/7-sending-notifications.md:101]: @justinedelson it completely breaks the parser. Going forward we'll have some pretty nicely styled code blocks. See example: https://developers.corp.adobe.com/parliament-demo/segments.md#terms

Comment (justinedelson) [reply tutorial/7-sending-notifications.md:101]: So should all of the "override" style blocks be removed? I'll have to doublecheck how removing these impacts the current rendering -- I had some complicated setup with Charles working before, but haven't done that since reimgaging my laptop.

Comment (macdonst) [reply tutorial/7-sending-notifications.md:101]: @justinedelson weirdly that was the only one it complained about. We can wait until post MAX timeframe to merge this. I was just trying to get ahead of the game.

Comment (justinedelson) [reply tutorial/7-sending-notifications.md:101]: Didn't seem to be a problem.
